enzyme/tools/enzyme-tblgen/enzyme-tblgen.cpp

```diff
@@ -1203,10 +1166,30 @@ void handleUse(
   bool usesShadow = Def->getValueAsBit("usesShadow");
   bool usesCustom = Def->getValueAsBit("usesCustom");
 
-  // We don't handle any custom primal/shadow
   (void)usesCustom;
   assert(!usesCustom);
 
+  if (Def->isSubClassOf("StaticSelect")) {
```

Comment:
these should be in the other order the assertion come after the handler, and the handler should also return to ensure the error isn't hit

Reply:
ah right, I moved the assert after and set StaticSelect to have useCustom

Reply:
I realized a bug in the useCustom analysis logic that was easier to just code up so I went ahead and fixed it